src/__tests__/index.test.js

```diff
@@ -2,17 +2,35 @@ const prometheus = require('../index');
 
 test('Should allow direct connections', () => {
     const next = jest.fn();
-    prometheus.middleware({
+    prometheus.middleware()({
         headers: {},
         state: {},
     }, next);
     expect(next.mock.calls.length).toBe(1);
 });
 
-test('Should allow direct connections to /metrics', () => {
+test('Should return metrics for overriden path', () => {
     const next = jest.fn();
     const vals = {};
     prometheus.middleware({
+        path: '/test'
+    })({
+        headers: {},
+        state: {},
+        path: "/test",
+        set: (key, value) => {
+           vals[key] = value;
+        }
+    }, next);
+    expect(next.mock.calls.length).toBe(1);
+    expect(Object.keys(vals).includes('Content-Type'));
```

Comment:
This should be `expect(Object.keys(vals).includes('Content-Type')).toBe(true);`. Without the `toBe(true)` at the end no assertion is running.